src/nimodinst/templates/session.py.mako

```diff
@@ -143,9 +143,9 @@ class Session(object):
 
     def close(self):
         # TODO(marcoskirsch): Should we raise an exception on double close? Look at what File does.
-        if(self.handle != 0):
-            self._close_installed_devices_session(self.handle)
-            self.${config['session_handle_parameter_name']} = 0
+        if(self._handle != 0):
+            self._close_installed_devices_session(self._handle)
```

Comment:
These two lines should use `${config['session_handle_parameter_name']}`

Reply:
fixed